csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -300,10 +296,7 @@ public bool Remove(T item)
         /// <param name="values">The values to add to this collection.</param>
         public void Add(RepeatedField<T> values)
         {
```

Comment:
Just wondering why not provide AddRange(RepeatedField<T> values) like IEnumerable<T> values?